frontend/src/App.tsx

```diff
@@ -1,11 +1,11 @@
 import { useEffect, useState } from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import './App.css';
-import Authentication from 'pages/authentication/Authentication';
-import useFetch from 'components/use-fetch/useFetch';
-import { authReq } from 'services/AuthenticationService';
-import Feed from 'pages/feed/Feed';
-import Header from 'components/header/Header';
+import Authentication from './pages/authentication/Authentication';
+import useFetch from './Components/use-fetch/useFetch';
+import { authReq } from './Services/AuthenticationServices';
+import Feed from './pages/feed/Feed';
+import Header from './Components/Header/Header';
```

Comment:
May be better to import using not relativ path such as "Components/Header/Header" instead of "./Components/Header/Header"